spinedb_api/db_mapping_base.py

```diff
@@ -235,17 +259,26 @@ def _add_descendants(self, item_types):
             if not changed:
                 break
 
-    def get_mapped_item(self, item_type, id_, fetch=True):
+    def reset_purging(self):
+        """Resets purging status for all item types.
+
+        Fetching items of an item type that has been purged will automatically mark those items removed.
+        Resetting the purge status lets fetched items to be added unmodified.
+        """
+        for mapped_table in self._mapped_tables.values():
+            mapped_table.wildcard_item.status = Status.committed
+
+    def get_mapped_item(self, item_type, id_):
         mapped_table = self.mapped_table(item_type)
-        return mapped_table.find_item_by_id(id_, fetch=fetch) or {}
+        return mapped_table.find_item_by_id(id_) or {}
 
```

Comment:
Removing the fetch argument here makes me nervous. Was it never used? Or what was the reasoning?

Reply:
It was never used. The `get_mapped_item()` method is only called from `export_functions` which expects it to fetch.